app/src/main/java/io/intrepid/skeleton/base/BaseView.java

```diff
@@ -0,0 +1,4 @@
+package io.intrepid.skeleton.base;
```

Comment:
It's empty now, but there's a good chance that in the future people want to add methods that all views should expect to have.